### PR TITLE
Update year in LICENSE

### PR DESCRIPTION
**Pull request by hedyhli**
Happy belated new year :tada:

2018 -> 2018-2020

I don't think much description is needed 

If this is merged I will also update the licenses in the other repos (unless someone does it before me)

### CODE REVIEW

**Comment by ghost** [comment]
Thank you for contributing to Python Discord!

Please check out the following documents:
- [Join the Discord server!](https://discord.gg/python) - If you have not already, join the Discord server and introduce yourself in our #dev-contrib channel.
- [Python Discord Contributor Wiki](https://pythondiscord.com/pages/contributing/) - A set of resources about setting up our services
- [Python Discord Code of Conduct](https://pythondiscord.com/pages/code-of-conduct) - Code of conduct for our community
- [CONTRIBUTING.md](https://github.com/python-discord/site/blob/master/CONTRIBUTING.md) - Guidelines on contributing to this repository

File: LICENSE
Comment (Akarys42) [inline LICENSE]: Let's make it inclusive 
```suggestion
Copyright (c) 2018-2020 Python Discord
```

**Review by Akarys42 — CHANGES_REQUESTED** [review]
This is fine by me, but it is better if we make it inclusive (`2018-2020` instead of just `2020`).

Once this is done, we can merge it, but for the other repo one of the core devs will directly push to master, it will be easier this way. Thank you!

**Comment by hedyhli** [comment]
It might be neater to squash the commits but I'm not on my computer at the moment

**Review by Akarys42 — APPROVED** [review]

**Review by l33tlinuxh4x0r — APPROVED** [review]
lol we missed 2019 and it is almost time for 2021 but looks good.

**Review by jb3 — APPROVED** [review]

**Pull request merged**
